app/controllers/chef_ingredients_controller.rb

```diff
@@ -0,0 +1,8 @@
+class ChefIngredientsController < ApplicationController
+    def index
+        @chef = Chef.find(params[:chef_id])
+        @dishes = @chef.dishes
```

Comment:
see Chef model for a note about how you can grab the chef ingredients more easily by using an additional association